### PR TITLE
0.15 release notes

### PR DESCRIPTION
**Pull request by jni**
## Description

I've made a branch for 0.15. This is the initial draft for the release notes. I've used @hmaarrfk's script to automatically generate the release notes, then manually moved PRs around to the different sections as needed, roughly speaking. Comments welcome!

### CODE REVIEW

File: doc/release/release_0.15.rst [hunk 1]
```diff
@@ -0,0 +1,556 @@
+Announcement: scikit-image 0.15.0
+=================================
+
+We're happy to announce the release of scikit-image v0.15.0! This one's a
+treat. ;)
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space mani pulation,
```
Comment (hmaarrfk) [inline doc/release/release_0.15.rst]:
```suggestion
for segmentation, geometric transformations, color space manipulation,
```

File: doc/release/release_0.15.rst [hunk 2]
```diff
@@ -0,0 +1,556 @@
...
+- Use skimage gaussian filter to avoid integer rounding artifacts (#2983)
+- Remove Python 2 compatibility (#3000)
+- Add ``rectangle_perimeter`` feature to ``skimage.draw`` (#3069)
+- Use proper axis indexing in ``adapt_rgb`` ``each_channel`` (#3097)
+- Linking preserve_range parameter calls to docs (#3109)
+- Update installation instructions to reference existing requirements specification (#3113)
+- Updated release notes with pre 0.13.1 phase (#3114)
+- Release guidelines update (#3115)
+- Ensure we are installing with / running on Python 3 (#3119)
+- Prefer imageio over PIL for file manipulations. (#3126)
```
Comment (sciunto) [inline doc/release/release_0.15.rst]: might be highlighted

File: doc/release/release_0.15.rst [hunk 3]
```diff
@@ -0,0 +1,556 @@
...
+- More explicit example title, better list rendering in plot_cycle_spinning.py (#3606)
+- Add rgb to hsv example in the gallery (#3607)
+- Update documentation of ``perimeter`` and add input validation (#3608)
+- Additionnal mask option to clear_border (#3610)
+- Set up CI with Azure Pipelines (#3612)
+- [MRG] EHN: median filters will accept floating image (#3616)
+- Update Travis-CI to xcode 10.1 (#3617)
+- Minor tweaks to _mean_std code (#3619)
+- Reduce the default tolerance in threshold_li (#3622)
+- Added glossary to the doc (#3626)
```
Comment (sciunto) [inline doc/release/release_0.15.rst]: might be highlighted

File: doc/release/release_0.15.rst [hunk 4]
```diff
@@ -0,0 +1,556 @@
...
+- Minor PEP8 fixes (#3305)
+- cython: check for presence of cpp files during install from sdist (#3311)
+- Remove deprecated ``dynamic_range`` in ``measure.compare_psnr`` (#3313)
+- appveyor: don't upload any artifacts (#3315)
+- Add benchmark suite for hough_line() (#3319)
+- Novice skip url test (#3320)
+- Remove benchmarks from wheel (#3321)
+- Add license file to the wheel (binary) distribution (#3322)
+- codecov: ignore build scripts in coverage and don't comment on PRs (#3326)
+- Correct rotate method's center parameter doc (#3341)
```
Comment (sciunto) [inline doc/release/release_0.15.rst]: might be highlighted

File: doc/release/release_0.15.rst [hunk 5]
```diff
@@ -0,0 +1,556 @@
...
+- Release guidelines update (#3115)
+- Ensure we are installing with / running on Python 3 (#3119)
+- Prefer imageio over PIL for file manipulations. (#3126)
+- Update the documentation regarding datalocality (#3127)
+- Hide warnings in test_unsharp_mask (#3130)
+- Process 0.15 deprecations (#3132)
+- Documentation: always use dev branch javascript (#3136)
+- Add initial airspeed velocity (asv) framework (#3137)
+- Supress warnings for flatten during io testing (#3143)
+- Bugfix separate stains log10 (#3146)
```
Comment (sciunto) [inline doc/release/release_0.15.rst]: might be highlighted

File: doc/release/release_0.15.rst [hunk 6]
```diff
@@ -0,0 +1,556 @@
...
+- Update ISSUE_TEMPLATE.md with info about numpy and skimage versions (#3730)
+- Use relative imports for many cython modules (#3759)
+- Pass tests that don't raise floating point exceptions on arm with soft-fp (#3337)
+- ENH: add range option for histogram. (#2479)
+- BUG: Fix greycoprops correlation always returning 1 (#2532)
+- Add section on API discovery via ``skimage.lookfor`` (#2539)
+- Better document remove_small_objects behaviour: int vs bool (#2830)
+- Speedup 2D warping for affine transformations (#2902)
+- Credit Reviewers in Release Notes (#2927)
+- Added small galleries in the API (#2940)
```
Comment (sciunto) [inline doc/release/release_0.15.rst]: might be highlighted

File: doc/release/release_0.15.rst [hunk 7]
```diff
@@ -0,0 +1,556 @@
...
+
+
+Other contributions and improvements
+------------------------------------
+- MNT: handle a deprecation warning for np.linspace and floats for the num parameter (#3453)
+- TST: numpy empty arrays are not inherently Falsy (#3455)
+-  handle warning in scipy cdist for unused parameters (#3456)
+- MNT: don't use filter_warnings in test suite. (#3459)
+- Add doc notes on setting up the build environment (#3472)
+- Release the GIL in numerous cython functions (#3490)
```
Comment (sciunto) [inline doc/release/release_0.15.rst]: might be highlighted

File: doc/release/release_0.15.rst [hunk 8]
```diff
@@ -0,0 +1,556 @@
...
+- make the low contrast check optional when saving images (#3653)
+- Correctly ignore release notes auto-generated for docs (#3656)
+- Remove MANIFEST file when making the 'clean' target (#3657)
+- Clarify return values in _overlap docstrings in feature/blob.py (#3660)
+- Contribution script: allow specification of GitHub development branch (#3661)
+- Update core dev guide: deprecation, contributor guide, required experience (#3662)
+- Add release notes for 0.14.2 (#3664)
+- FIX gallery: Add multichannel=True to match_histogram (#3672)
+- MAINT Minor code style improvements (#3673)
+- Pass parameters through tifffile plugin (#3675)
```
Comment (soupault) [inline doc/release/release_0.15.rst]: I'd shortlist this.

File: doc/release/release_0.15.rst [hunk 9]
```diff
@@ -0,0 +1,556 @@
...
+- Build tools: Downgrade Xcode to 9.4 on master (#3723)
+- Improve visual_test.py (#3732)
+- Updated painttool to work with color images and properly scale labels. (#3733)
+- Add image.sc forum badge to README (#3738)
+- Blacklist PyQt 5.12.0 on Travis (#3743)
+- Build tools: Fix matplotlib + qt 5.12 the same way upstream does it (#3744)
+- Add image of retina to our data (#3748)
+- gallery: remove xx or yy  sorted directory names (#3761)
+- Allow for f-contiguous 2D arrays in convex_hull_image (#3762)
+- Add microaneurysms() to gallery (#3765)
```
Comment (soupault) [inline doc/release/release_0.15.rst]: -> New features.

File: doc/release/release_0.15.rst [hunk 10]
```diff
@@ -0,0 +1,556 @@
...
+- Fix potential memory leak (#3718)
+- Fix potential use of NULL pointers (#3719)
+- Fix and improve core_cy.pyx (#3720)
+- Build tools: Downgrade Xcode to 9.4 on master (#3723)
+- Improve visual_test.py (#3732)
+- Updated painttool to work with color images and properly scale labels. (#3733)
+- Add image.sc forum badge to README (#3738)
+- Blacklist PyQt 5.12.0 on Travis (#3743)
+- Build tools: Fix matplotlib + qt 5.12 the same way upstream does it (#3744)
+- Add image of retina to our data (#3748)
```
Comment (soupault) [inline doc/release/release_0.15.rst]: -> New features.

File: doc/release/release_0.15.rst [hunk 11]
```diff
@@ -0,0 +1,556 @@
...
+for segmentation, geometric transformations, color space mani pulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+https://scikit-image.org
+
+0.15 is the first scikit-image release that is only compatible with Python 3.5
+and above. Python 2.7 users should strongly consider upgrading to Python 3.5+,
+or use the 0.14 long term support releases.
```
[the remark this answers is not in the document]
Comment (hmaarrfk) [reply doc/release/release_0.15.rst]: I prefer the original wording.

**Pull request closed without merging**